### DISCUSSION TITLE
please upgrade bundled kubectl version

### DISCUSSION THREAD

**Discussion by rtomadpg** [helmfile/helmfile#1510] [Ideas]
Please consider upgrading the bundled version of kubectl, as v1.25.16 is quite old.

Ref: https://github.com/helmfile/helmfile/blob/f09855d40e401beab96c875d0d3bbe43b222413f/Dockerfile#L53

**Comment by yxxhero** [comment]
PR is welcome.

**Comment by rtomadpg** [comment]
Hi @yxxhero - sure thing. Here is https://github.com/helmfile/helmfile/pull/1516
